flexibleengine/resource_flexibleengine_cce_cluster_v3.go

```diff
@@ -289,13 +289,17 @@ func resourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.SetId(n.Metadata.Id)
 	d.Set("name", n.Metadata.Name)
 	d.Set("status", n.Status.Phase)
+	d.Set("labels", n.Metadata.Labels)
+	d.Set("annotations", n.Metadata.Annotations)
```

Comment:
lables and annotations not included in the cluster API response, so this Set will make them empty :(

Reply:
thanks for your review. I edited the code accordingly to the api response